CMakeLists.txt

```diff
@@ -48,6 +48,9 @@ if(MSVC)
     if(WARNINGS_AS_ERRORS)
         add_compile_options(/WX)
     endif()
+
+    # Suppress warnings about std::getenv
+    add_compile_definitions(-D_CRT_SECURE_NO_WARNINGS)
```

Comment:
Why is this necessary?

Reply:
This was added because the Windows CI builder claims that `std::getenv` is unsafe. It isn't, in this case, since we are just reading and immediately using the thread limit env variable.